src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,428 @@
+compression, and to avoid unnecessary conversions of the original data from a non-tiled to a
+tiled format, or vice-versa.
+
+For BIDS, Microscopy raw data MUST be stored in one of the following formats:
+
+-   [Portable Network Graphics](http://www.libpng.org/pub/png/) (`.png`)
+
+-   [Tag Image File Format](https://www.adobe.io/open/standards/TIFF.html) (`.tif`)
+
+-   [OME-TIFF](https://docs.openmicroscopy.org/ome-model/6.1.2/ome-tiff/specification.html#)
+    (`.ome.tif` for standard TIFF files or `.ome.btf` for
+    [BigTIFF](https://www.awaresystems.be/imaging/tiff/bigtiff.html) files)
+
+If different from PNG, TIFF or OME-TIFF, the original unprocessed data in the native format MAY be
+stored in the [`/sourcedata` directory](../02-common-principles.md#source-vs-raw-vs-derived-data).
+
+Future versions may extend this list of supported file formats.
+For example with the [OME-NGFF](https://ngff.openmicroscopy.org/latest/) format currently
+developed by OME as a successor to OME-TIFF for better remote sharing of large datasets.
+
+### Modality suffixes
+Microscopy data currently support the following imaging modalities:
+<!---
+The following table will be generated automatically with macros after community review.
+A "manual" table is provided to facilitate the review process.
+-->
+| **Name**                                      | `suffix` |
+| --------------------------------------------- | -------- |
+| Transmission electron microscopy              | TEM      |
+| Scanning electron microscopy                  | SEM      |
+| Micro-CT                                      | uCT      |
+| Bright-field microscopy                       | BF       |
+| Dark-field microscopy                         | DF       |
+| Phase-contrast microscopy                     | PC       |
+| Differential interference contrast microscopy | DIC      |
+| Fluorescence microscopy                       | FLUO     |
+| Confocal microscopy                           | CONF     |
+| Polarized-light microscopy                    | PLI      |
+| Coherent anti-Stokes Raman spectroscopy       | CARS     |
+| 2-photon excitation microscopy                | 2PE      |
+| Multi-photon excitation microscopy            | MPE      |
+| Super-resolution microscopy                   | SR       |
+| Nonlinear optical microscopy                  | NLO      |
+| Optical coherence tomography                  | OCT      |
+| Selective plane illumination microscopy       | SPIM     |
+
+### Filename entities
+
+In the context of Microscopy, a session ([`ses-<label>`](../99-appendices/09-entities.md#ses))
+can refer to all the acquisitions between the start and the end of an imaging experiment
+for ex vivo imaging or a subject lab visit for biopsy procedure and/or in vivo imaging.
+
+The [`sample-<label>`](../99-appendices/09-entities.md#sample) entity is REQUIRED for
+Microscopy data and is used to distinguish between different samples from the same subject.
```

Comment:
how unfeasible to have a single sample from a subject?  
I just wonder why it is not an OPTIONAL, 
- if it is possible to have a single sample, why should I bother to come up with a label for it? (most likely matching the `subject`)
- like e.g. `run` or even `session` - introduced/used whenever they are the best way to disambiguate between different "data samples"

Reply:
This is a great question!
It goes back to the discussion about subject vs. sample in https://github.com/bids-standard/bids-specification/issues/779, and other early discussions on the BEP google-doc.

We needed to preserve the meaning of `subject` of BIDS as "a person or animal participating in the study" whereas the `sample` is the sample (slice, tissue, etc) itself under the microscope.
The distinction was necessary to avoid ambiguity and ensure consistency between modalities (ex: a  `subject` in `anat` always refer to the same "physical entity" as a `subject` in `microscopy`).

For the same reasons, samples' attributes such as "sample type" are described in a separate tsv file (samples.tsv) than subjects' attributes (participants.tsv).